offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/person/update/UpdatePersonView.groovy

```diff
@@ -32,6 +38,69 @@ class UpdatePersonView extends CreatePersonView{
     private void adjustViewElements() {
         submitButton.caption = "Update Person"
         abortButton.caption = "Abort Person Update"
+
+        //add a grid
+        affiliations = new Grid<>()
+        generateAffiliationGrid()
+        this.addComponent(affiliations,2)
```

Comment:
I think having a caption would make sense to indicate what is shown in the grid. 
```suggestion
        affiliations.setCaption("Current Affiliations")
        this.addComponent(affiliations,2)
```